deployments/azure-staticwebapp.yml

```diff
@@ -52,6 +58,8 @@ jobs:
           ###### End of Repository/Build Configurations ######
 
   close_pull_request_job:
+    permissions:
+      contents: none
```

Comment:
Closing pull requests doesn't require any permissions?

Reply:
@Phantsure In `close_pull_request` job, github_token is not supplied as `action_input`. As the result [tool](https://app.stepsecurity.io) added `contents:none`.
The token from the previous job can't be used in this job, as it expires, when previous job finishes. [check](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret)
I also checked [action.yml](https://github.com/Azure/static-web-apps-deploy/blob/v1/action.yml#L31) of Azure/static-web-apps-deploy.  `Token` is used only for commenting on pull-requests.

So in this job maybe we have add `repo-token`. What do you think ?

Thanks :)

Reply:
I think if `repo_token` was required then partner Azure team would have added that. Deployment templates are mostly maintained by partners like azure, gcp, aws, etc so they would be able to better comment on this.